tests/sentry/event_manager/grouping/test_assign_to_group.py

```diff
@@ -524,12 +505,9 @@ def test_uses_regular_or_optimized_grouping_as_appropriate(
         patch("sentry.grouping.ingest.config.is_in_transition", return_value=in_transition),
         override_options({"grouping.config_transition.killswitch_enabled": killswitch_enabled}),
     ):
-        config = MOBILE_CONFIG if mobile_config else NEWSTYLE_CONFIG
-        save_event_with_grouping_config({"message": "Dogs are great!"}, project, config)
+        save_event_with_grouping_config({"message": "Dogs are great!"}, project, NEWSTYLE_CONFIG)
 
-    if mobile_config or killswitch_enabled:
-        assert mock_save_aggregate.call_count == 1
-    elif flag_on:
```

Comment:
Same request here as in https://github.com/getsentry/sentry/pull/76998#discussion_r1747619413 - can we please keep the killswitch for the moment?